.github/workflows/create_rc.yml

```diff
@@ -0,0 +1,150 @@
+        id: determine_commit
+        run: |
+          if [ "${{ github.event.inputs.rc_commit_sha }}" == "" ]; then
+            echo "No commit sha provided, using the latest commit"
+            echo "commit_sha=$(git rev-parse HEAD)" >> $GITHUB_ENV
+          else
+            echo "Using provided commit sha"
+            echo "commit_sha=${{ github.event.inputs.rc_commit_sha }}" >> $GITHUB_ENV
+          fi
+
+      - name: Create release candidate tag
+        id: create_release
+        run: |
+
+          git config --global user.email "rc_githubactions@github.com"
+          git config --global user.name "Release Candidate GitHub Action "
+
+          git reset --hard ${{ env.commit_sha }}
+
+          # Use standard-version to determine the next version
+          npm run release -- --skip.commit --skip.tag --skip.changelog
+          new_version=$(jq -r '.version' package.json)
+          branch_name=rc/v${new_version%.*}
+          echo "rc_branch=$branch_name" >> $GITHUB_OUTPUT
+          release_branch_name=${branch_name//rc/release}
+
+          git reset --hard ${{ env.commit_sha }}
+
+          if [ "${{ github.event.inputs.dry_run }}" == "true" ]; then
+            echo "Dry run enabled. The following commands would be executed:"
+            echo "git checkout -b $branch_name"
+            echo "npm run release -- --prerelease rc --skip.changelog --dry-run"
+            npm run release -- --prerelease rc --skip.changelog --dry-run
+            echo "git push -u origin $branch_name --follow-tags"
+          else
+            
+            git checkout -b $branch_name
+            npm run release -- --prerelease rc --skip.changelog
+            git push -u origin $branch_name --follow-tags
+          fi
+
+      - name: Create Github Release
+        id: create_github_release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          tag=$(jq -r '.version' package.json)
+          if [ "${{ github.event.inputs.dry_run }}" == "true" ]; then
+            echo "Dry run enabled. The following commands would be executed otherwise:"
+            echo gh release create "$tag" --title="Release Candidate: $tag" --notes="Automated release for $tag" --latest=false -p
+          else
+            gh release create "$tag" --title="Release Candidate: $tag" --notes="Automated release for $tag" --latest=false -p
+          fi
+
+  chart-release:
+    needs: create_patch_release
+    if: always() && contains(needs.create_patch_release.result, 'success') && !github.event.act
+    runs-on: ubuntu-22.04
+    container:
+      image: linode/apl-tools:v2.8.2
+      options: --user 0 # See https://docs.github.com/en/actions/sharing-automations/creating-actions/dockerfile-support-for-github-actions#user
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ needs.create_patch_release.outputs.rc_branch }}
+
+      - name: Prepare chart
```

Comment:
Is this copy-paste from .github/workflows/main.yml file?
Is so then we should also add it as script to ci/ directory and refer to it in this pipeline